streampark-console/streampark-console-service/src/main/assembly/script/schema/mysql-schema.sql

```diff
@@ -189,8 +189,9 @@ create table `t_flink_project` (
   `url` varchar(255) collate utf8mb4_general_ci default null,
   `branches` varchar(64) collate utf8mb4_general_ci default null,
   `user_name` varchar(64) collate utf8mb4_general_ci default null,
-  `password` varchar(64) collate utf8mb4_general_ci default null,
+  `password` varchar(512) collate utf8mb4_general_ci default null,
```

Comment:
512 😳, I think this length is too long? Can you confirm it?

Reply:
This situation is in response to very long passwords, and it cannot be ruled out that there are users whose passwords are very long